server/controllers/finance/reports/vouchers/receipt.handlebars

```diff
@@ -10,7 +10,6 @@
       </div>
     {{/if}}
   {{/header}}
-  QQQ1
```

Comment:
Oh la la.  Who do you suppose uses vim around here?  😆

Reply:
No, this one was mine!  DEFINITELY unrelated to vim!